miniwallet/deploy/001_deploy_miniWallet.ts

```diff
@@ -0,0 +1,71 @@
+import config from '../config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ethers, upgrades } from 'hardhat'
+
+// import config from '../config'
+
+const OPERATOR_ROLE = ethers.utils.id('OPERATOR_ROLE')
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  let initialOperatorThreshold
+  let initialOperators
+  let initialUserLimit
+  let initialAuthLimit
+
+  if (hre.network.config.chainId === 31337) {
+    console.log('Local Testing Deploy')
+    initialOperatorThreshold = config.test.initialOperatorThreshold
+    initialOperators = config.test.initialOperators
+    initialUserLimit = config.test.initialUserLimit
+    initialAuthLimit = config.test.initialAuthLimit
+  } else {
+    initialOperatorThreshold = config.initialOperatorThreshold
+    initialOperators = config.initialOperators
+    initialUserLimit = config.initialUserLimit
+    initialAuthLimit = config.initialAuthLimit
+  }
+
+  console.log('operators:', JSON.stringify(config.initialOperators))
+  const MiniWallet = await ethers.getContractFactory('MiniWallet')
+  const miniWallet = await upgrades.deployProxy(
```

Comment:
need to know the source code of the contract being used here, and to record the addresses of proxy contract and logic contract (and version, time)

Reply:
Tracking in #13 and [here](https://github.com/polymorpher/sms-wallet/blob/miniV0/miniwallet/PROXY.md#improvements)